pkg/reconciler/v1alpha1/subscription/subscription.go

```diff
@@ -18,33 +18,36 @@ package subscription
 
 import (
 	"context"
+	"encoding/json"
```

Comment:
Let's not put this file into the v1alpha1. I was curious about this and chatted with @mattmoor about this and the Serving plan is to move the reconcilers out of the version specific directories, as expected, the reconciler should NOT be version specific, since they operate on the at-rest resources. So, since they are planning on moving them, let's not add these there and then have to move them out of there in a follow on. Especially since this is the first one of these, let's do it right so the follow on ones are then using the correct path.

Reply:
I moved it.